app/models/document.rb

```diff
@@ -21,9 +21,29 @@ class Document < ApplicationRecord
   belongs_to :submission
 
   def external_url
+    if ENV["RAILS_ENV"] == "production" || ENV["RAILS_ENV"] == "staging"
```

Comment:
I think the standard way to do this is `Rails.env.production? || Rails.env.staging?`